tests/load/pipeline/test_model_item_format.py

```diff
@@ -0,0 +1,170 @@
+# test the sql insert job loader, works only on duckdb for now
```

Comment:
I am not quite sure what else to test here, basically we are just testing wether these sql statements are created properly, all the column types etc should work since we are on the same destination and if they don't, there is not much we can do about it really, since we are not touching the data.